src/main/java/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -0,0 +1,78 @@
+package competition.subsystems.collector;
+
+import competition.electrical_contract.ElectricalContract;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANSparkMax;
+import xbot.common.controls.actuators.XSolenoid;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class CollectorSubsystem extends BaseSubsystem {
+    public XCANSparkMax collectorMotor;
+    public XSolenoid collectorSolenoid;
+    public DoubleProperty intakePower;
+    public DoubleProperty ejectPower;
+    final ElectricalContract contract;
+
+    public enum collectorState{
```

Comment:
change `collectorState` to `CollectorState` to match overall code style

Reply:
change made